library/blivet.py

```diff
@@ -205,6 +203,56 @@ def _manage_one_encryption(self, device):
 
         return ret
 
+    def _new_mdarray(self, members, fmt, raid_name=""):
```

Comment:
Both places you call this you are passing `self._get_format()` as `fmt`, so I think you can remove that parameter.

Reply:
Fixed: Removed `fmt` parameter. Value is now obtained from `self`.